### PR TITLE
[ADD] Adds product_code_cpv module.

### PR DESCRIPTION
**Pull request by daramousk**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Maybe this should go to OCA/community-data-files, and if this is a standard nomenclature, include as data the records for the given nomenclature.

File: product_code_cpv/__init__.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 bitwise.solutions <https://bitwise.solutions>
```
Comment (NL66278) [inline product_code_cpv/__init__.py:2]: No copyright in init files.

File: product_code_cpv/__init__.py [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
```
Comment (NL66278) [inline product_code_cpv/__init__.py:3]: https: on license files.

File: product_code_cpv/models/product_template_cpv.py [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 bitwise.solutions <https://bitwise.solutions>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo import fields, models
+
+
+class ProductTemplateCPV(models.Model):
+    _name = 'product.template.cpv'
+
+    name = fields.Char()
```
Comment (NL66278) [inline product_code_cpv/models/product_template_cpv.py:10]: All fields should be both required (therefore not NULLable in SQL) and readonly.

**Review by NL66278 — CHANGES_REQUESTED** [review]
:+1: Nice addition, but some changes would be welcome. See other comments. And you might look into Pedro's suggestion on the right repository.

**Pull request closed without merging**